### DISCUSSION TITLE
Q: 请问评论中的图片如何实现点击放大预览功能呢？ || Q: May I ask how to realize the click-to-enlarge preview function for the pictures in the comments?

### DISCUSSION THREAD

**Discussion by MarkRushB** [walinejs/waline#2044] [Q&A] [answered]
### Describe the bug | 问题描述

如题！Waline非常好用，想用来搭建一个自己的微博的服务，想问一下评论中的图片支不支持点击预览呀？类似于弹出一个lightbox的样子，如果可以的该如何配置呢？

### Website URL | 问题网站

https://waline-blue-one.vercel.app/

### Where your waline deploy? | 服务部署在哪里？

Vercel (Default)

### Where your comment data store? | 数据存储在哪里？

LeanCloud(https://leancloud.app)

---
### Describe the bug | Problem description

As the title! Waline is very easy to use. I want to use it to build my own Weibo service. I would like to ask if the pictures in the comments support clicking to preview? Similar to popping up a lightbox, if possible, how to configure it?

### Website URL | Problem Website

https://waline-blue-one.vercel.app/

### Where your waline deploy? | Where is the service deployed?

Vercel (Default)

### Where your comment data store? | Where is your comment data store?

LeanCloud(https://leancloud.app)

**Comment by lizheming** [comment, marked as answer]
https://waline.js.org/cookbook/customize/image-preview.html

**Reply by MarkRushB** [reply]
多谢！！

---
thanks! !
